configure.ac

```diff
@@ -76,6 +76,42 @@ AS_IF([test "x$enable_convert" != "xno"], [
 	AC_DEFINE([ENABLE_CONVERT], [0])
 ])
 
+AC_ARG_ENABLE([barebox],
+	AS_HELP_STRING([--disable-barebox], [Disable support for barebox bootloader])
```

Comment:
The useful cases are probably:

- all bootloader
- a specific bootloader
- no bootloader

Can we handle that nicely in configure? If it's not obvious, maybe drop this change from this PR, so we can look at it separately.

Reply:
I'm not very familiar with autoconf, but perhaps there's some standard idiom that would allow a `--enable-bootloaders=barebox,u-boot`, with `all` a keyword with the obvious meaning and being default. That would at least reduce the ./configure UI complexity. I'll take a closer look and drop it for the time being if there's no reasonably simple solution.

Reply:
@michaelolbrich Do you know something that would help here?

Reply:
`--enable-bootloaders=` is probably the best from a user POV, but I'm not aware of any helpers to implement this.
And I'd also accept `yes`/`no` for all or nothing as we..

Reply:
`no` would only be useful for the host, and we don't really need to save space there. So maybe drop it from this PR to get the rest merged, @Villemoes?

Reply:
Yeah, that was what I was planning to do anyway in order not to conflict with the "custom bootloader backend" proposal. I updated my branch locally last night, will rebase and push a new one once the 'rm-checksum' PR is merged shortly.